lib/model.js

```diff
@@ -758,6 +758,14 @@ module.exports = function(registry) {
     var pathName =
       (scope.options && scope.options.http && scope.options.http.path) || scopeName;
 
+    var updateOnlyProps = this.getUpdateOnlyProperties();
+    var createOnly = false;
```

Comment:
@bajtos I have two environment 1) loopback application where I can test with the API designer end to end 2) my git hub branch where I can do run tests and checkins etc. I addressed above comments in the first env to make sure I test end-to-end through API Explorer with these code review comment changes and forgot to do same change in 2nd env/local git hub branch. Tests were passing since my previous code was logically correct. Anyway, done the above change now...